build/build.ps1

```diff
@@ -87,8 +82,52 @@ function LocateVisualStudio {
     [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
     Invoke-WebRequest "http://github.com/Microsoft/vswhere/releases/download/$vswhereVersion/vswhere.exe" -OutFile $vswhereExe
   }
+  
+  return $vsWhereExe
+}
+
+function GetVSIXExpInstallerExe{
+  $vsixExpInstallerVersion = GetVersion("VSIXExpInstallerVersion")
+  $vsixExpInstalleDir = Join-Path $ToolsRoot "VSIXExpInstaller\$vsixExpInstallerVersion"
+  $vsixExpInstalleExe = Join-Path $vsixExpInstalleDir "tools\VSIXExpInstaller.exe"
+  
+  if (!(Test-Path $vsixExpInstalleExe)) {
+    Create-Directory $vsixExpInstalleDir
+    Write-Host "Downloading VSIXExpInstaller"
+    [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+    Invoke-WebRequest "https://dotnet.myget.org/F/roslyn-tools/api/v2/package/RoslynTools.VSIXExpInstaller/$vsixExpInstallerVersion" -OutFile RoslynTools.VSIXExpInstaller.zip
+    Expand-Archive .\RoslynTools.VSIXExpInstaller.zip -DestinationPath $vsixExpInstalleDir
```

Comment:
Why not include this in a toolset so that NuGet does the restore/expand/etc.

Reply:
I have no idea what a toolset is in this context.  Where is the documentation on that?  Note: we are not building or testing with the dotnet sdk today.  We use the version of MSBuild that comes with VS. We'd need to upgrade to a newer version of repotoolset to be VS independent